pkgs/development/interpreters/erlang/R17.nix

```diff
@@ -1,15 +1,19 @@
 { stdenv, fetchurl, perl, gnum4, ncurses, openssl
 , gnused, gawk, makeWrapper
 , odbcSupport ? false, unixODBC ? null
-, wxSupport ? false, mesa ? null, wxGTK ? null, xlibs ? null }:
+, wxSupport ? true, mesa ? null, wxGTK ? null, xlibs ? null
+, javacSupport ? false, openjdk ? null
```

Comment:
I'm happy with either just as long as someone tests if they can replicate on localhost and merge.